constants/requests.ts

```
@@ -68,3 +68,16 @@ export const INVALID_REQUEST_TYPE = "Invalid request type";
 export const INVALID_REQUEST_DEADLINE = "New deadline of the request must be greater than old deadline";
 export const REQUEST_UPDATED_SUCCESSFULLY = "Request updated successfully";
 export const UNAUTHORIZED_TO_UPDATE_REQUEST = "Unauthorized to update request";
+
+export const FEATURE_NOT_IMPLEMENTED="Feature not implemented"
```

ESLint

[error] Replace `="Feature·not·implemented"` with `·=·"Feature·not·implemented";`

(prettier/prettier)